ddht/v5_1/alexandria/sedes.py

```diff
@@ -43,5 +44,8 @@ def deserialize(self, value: bytes) -> bytes:
 AdvertiseSedes = List(AdvertisementSedes, max_length=32)
 AckSedes = Container(field_sedes=(uint256,))
 
+LocateSedes = Container(field_sedes=(content_key_sedes,))
+LocationsSedes = Container(field_sedes=(uint8, List(AdvertisementSedes, max_length=32)))
```

Comment:
Is `32` as a max number of advertisements to return is a limit that's up to the implementation to decide?

Reply:
It's derived from the maximum number of advertisements that *could* possibly fit into a packet.  Realistically, the max is probably about half of that number.